xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxBXA.h

```diff
@@ -56,17 +56,19 @@ class CDVDDemuxBXA : public CDVDDemux
 
   bool Open(CDVDInputStream* pInput);
   void Dispose();
-  void Reset();
-  void Abort();
-  void Flush();
-  DemuxPacket* Read();
+
+  // implementation of CDVDDemux
+  virtual void Reset() override;
+  virtual void Abort() override;
+  virtual void Flush() override;
+  virtual DemuxPacket* Read() override;
   bool SeekTime(double time, bool backwards = false, double* startpts = NULL) override { return false; }
-  void SetSpeed(int iSpeed) {};
-  int GetStreamLength() { return (int)m_header.durationMs; }
+  virtual void SetSpeed(int iSpeed) override {}
```

Comment:
You tell me :) You could make the argument that nearly any virtual method could have a trivial base class implementation. On the other hand, there might be value in being explicit by declaring the lack of any effect in its subclass.